### PR TITLE
feat: added dummy login and signup pages

### PR DESCRIPTION
**Pull request by RakshithBhat03**
# Dummy login and signup pages

## Note:
- Login and signup pages for visual purposes.
- Authentication will be implemented in future versions

## Preview: https://deploy-preview-4--clutchstore-react.netlify.app/

### CODE REVIEW

File: src/pages/Login/components/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                placeholder="Password"
+              />
+              <div className="auth-checkbox display-flex justify-content-between align-items-center">
+                <label className="display-flex align-items-center">
+                  <input type="checkbox" className="mr-5" />
+                  Remember me
+                </label>{" "}
+                <a href="/" className="txt-sm">
+                  Forgot Password?
+                </a>{" "}
```
Comment (ajay-mandaviya) [inline src/pages/Login/components/LoginForm.jsx]: remove a tag use Link

**Review by kedark152 — APPROVED** [review]
UI Looks very nice, code is written well.

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks fine 
code well structure 
just a give a small change

File: src/pages/Login/components/LoginForm.jsx (hunk 1 above)
Comment (RakshithBhat03) [reply src/pages/Login/components/LoginForm.jsx]: Thanks for pointing it out.

**Pull request merged**